src/VirtualList/VirtualList.js

```diff
@@ -7,22 +7,36 @@
 import DOMUtil from '../Util/DOMUtil';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import throttle from 'lodash.throttle';
+
+import BindMixin from '../Mixin/BindMixin';
 import Util from '../Util/Util';
 
 let mathMax = Math.max;
```

Comment:
I tested this out and actually had a significant impact when I implemented this in the first place. Let's create a separate task to optimise this